app/components/feeds/FeedSelectorItems.module.css

```
@@ -0,0 +1,5 @@
+.cardList {
```

Stylelint (17.2.0)

[error] Expected class selector ".cardList" to be kebab-case (selector-class-pattern)

(selector-class-pattern)